### ISSUE TITLE
problem of lemen_dataset.py

### ISSUE THREAD

**Issue by ScarletVV** [OHBA-analysis/osl#295]
When I ran this code, it output:
sub-032333 : **********************
sub-032333 : * PROCESSING FAILED! *
sub-032333 : **********************
sub-032333 : import_data : <function import_data at 0x7fd210b71900>
sub-032333 : <class 'FileNotFoundError'>
sub-032333 : [Errno 2] No such file or directory: '/home/uais1/Echo/OSL/DataPrepare/dataset/lemon/raw/sub-032333/RSEEG/sub-010041.vmrk'
sub-032333 : None

How can I fix this?

**Comment by cgohil8** [comment]
The problem is it can't find this file:
```
/home/uais1/Echo/OSL/DataPrepare/dataset/lemon/raw/sub-032333/RSEEG/sub-010041.vmrk
```
Seems like there's a mismatch between the filename and subject directory. It's trying to find the file `sub-010041.vmrk` in the directory for `sub-032333`.

**Comment by ScarletVV** [comment]
Yes it is, but my question is why the program wants to find the `sub-010041.vmrk` in the directory for `sub-032333` ?
I just download the data from [https://fcon_1000.projects.nitrc.org/indi/retro/MPI_LEMON/downloads/download_EEG.html](url) and use one of them(`sub-032333`) to test the code and 124 lines of code deleted
 ```
subjects = subjects[:2]
 ```
There is no subject named `sub-010041`  here, and no obvious reading of the .vmrk file code snippet in the code.

**Comment by cgohil8** [comment]
Where did you get `lemon_dataset.py` from? Can you send the contents of the script or a link.

**Comment by ScarletVV** [comment]
This is the result of Jupyter Notebook running https://github.com/ScarletVV/lemon_process.git.

**Comment by cgohil8** [comment]
What do you get when you print the `inputs` variable before you pass it to `preprocessing.run_proc_batch`?

**Comment by cgohil8** [comment]
What version of MNE are you using?

**Comment by ScarletVV** [comment]
MNE 1.3.1

**Comment by cgohil8** [comment]
Do get the same error if you do:
```
import mne

raw = mne.io.read_raw_brainvision("/home/uais1/Echo/OSL/DataPrepare/dataset/lemon/raw/sub-032333/RSEEG/sub-032333.vhdr")
```

**Comment by ScarletVV** [comment]
yes

**Comment by cgohil8** [comment]
ah, it's an error in MNE then. You can try upgrade MNE:
```
conda activate osl
pip install --upgrade mne
```
Although OSL hasn't been tested against versions above MNE 1.3.1, so you might run into new bugs. If you update MNE does the following work:
```
import mne

raw = mne.io.read_raw_brainvision("/home/uais1/Echo/OSL/DataPrepare/dataset/lemon/raw/sub-032333/RSEEG/sub-032333.vhdr")
```

**Comment by ScarletVV** [comment]
Same error after upgrading MNE.  Can I solve this problem if I install a lower version of MNE?

**Comment by ScarletVV** [comment]
Thank you for your patient guidance and answers. I will try similar work directly on MNE.

**Comment by cgohil8** [comment]
It's unlikely a lower version would fix this, but you can try. You might also be able to load it if you change the directory structure of the data. What happens if you create a new directory and copy only the `. vhdr` into it. E.g. you can try something like:
```
mkdir ~/Desktop/test
cp /home/uais1/Echo/OSL/DataPrepare/dataset/lemon/raw/sub-032333/RSEEG/sub-032333.vhdr ~/Desktop/test
python
>> import mne
>> raw = mne.io.read_raw_brainvision("~/Desktop/test/sub-032333.vhdr")
```
What happens then?

**Comment by ScarletVV** [comment]
Still same error...

**Comment by sunflower-smile** [comment]
The data we used is the LEMON dataset from the link you provided actually. 
Thanks a lot.

**Comment by cgohil8** [comment]
What version of MNE are you using?

**Comment by ScarletVV** [comment]
1.7.0 then down to 1.3.1 still same error.

**Comment by cgohil8** [comment]
Can you:
- Re-install OSL from scratch to make sure you have the latest code.
- Try commenting out the `set_eeg_reference` line in the config.
